src/components/Modal/__stories__/helpers.js

```diff
@@ -0,0 +1,25 @@
+export const animations = [
```

Comment:
I think that you can export all these const from the component, there is going to be a discrepancy as some point if we go this way :)